rosdep/python.yaml

```
@@ -438,6 +438,11 @@ pymodbustcp-pip:
   ubuntu:
     pip:
       packages: [pyModbusTCP]
+pynput:
+  ubuntu:
+    jammy: [python3-pynput]
+    noble: [python3-pynput]
+  debian: [python3-pynput]
```

This should generally match the name of the debian key.  Also, there is no reason to call out the Ubuntu names explicitly.  And finally, the distribution names need to be in alphabetical order.  So this should be:

```suggestion
python3-pynput:
  debian: [python3-pynput]
  ubuntu: [python3-pynput]
```

(you'll also need to move this around within this file to make the whole thing alphabetical)

---

> Focal for ubuntu doesn't host python3-pynput, so only have nobel and jammy specified. We can combine `pynput-pip` and `python3-pynput` and specify pip/non pip for specific releases, but will break backwards compatibility the the name 'pynput-pip'

OK, good point about Focal.  In that case, let's change this to be forwards-compatible:

```
  ubuntu:
    '*': [python3-pynput]
    focal: null
```